udata/core/dataset/forms.py

```diff
@@ -158,6 +172,19 @@ class DatasetForm(ModelForm):
         description=_("A short description of the dataset."),
     )
     license = fields.ModelSelectField(_("License"), model=License, allow_blank=True)
+    access_type = fields.SelectField(
+        choices=[(e.value, e.value) for e in AccessType],
+        default=AccessType.OPEN.value,
```

Comment:
`StrEnum` behaves as `str` (inherits from it) so you shouldn't need the value in places taking a `str`

```suggestion
        default=AccessType.OPEN,
```